code/modules/clothing/under/color.dm

```diff
@@ -9,7 +9,6 @@
 	item_state = "jumpsuit"
 	worn_icon_state = "jumpsuit"
 	worn_icon = 'icons/mob/clothing/uniform.dmi'
-	supports_variations = DIGITIGRADE_VARIATION
```

Comment:
This may need to be kept, you can modify the value down in the children of this typepath